wiki/Gameplay/Input_device/Touch_device/en.md

```diff
@@ -0,0 +1,6 @@
+# Touch Device
+
+A **touch device** is a display device that allows the user to interact with the computer by using their finger. 
+osu! supports touch devices nativley. You can use your finger to aim and click at the same time.
+
```

Comment:
```suggestion
If a touch device is used in osu!standard, an icon indicating usage of it would appear in the list of mods on the web. This mod is not shown in the stable client.
 
```

Reply:
is this shown in the "best performance" and "recent play" as a play mod?

Reply:
yep

Reply:
@mcendu I agree with this change, but "osu!standard" should be link to its stub